### PR TITLE
[ja] Translate tasks/administer-cluster/kubeadm/configure-cgroup-driver into Japanese

### LINKED ISSUE #31217: [ja] Translate tasks/administer-cluster/kubeadm/configure-cgroup-driver into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/administer-cluster/kubeadm/configure-cgroup-driver](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by ptux**
resolve #31217

### CODE REVIEW

File: content/ja/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: cgroupドライバーの設定
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+このページでは、kubeadmクラスタのコンテナランタイムcgroupドライバーに合わせて、kubelet cgroupドライバーを設定する方法について説明します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md:9]:
```suggestion
このページでは、kubeadmクラスターのコンテナランタイムcgroupドライバーに合わせて、kubelet cgroupドライバーを設定する方法について説明します。
```

same for below

File: content/ja/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+Kubernetesの[コンテナランタイムの要件](/docs/setup/production-environment/container-runtimes)を熟知している必要があります。
+
+<!-- steps -->
+
+## コンテナランタイムのcgroupドライバーの設定
+
+[Container runtimes](/docs/setup/production-environment/container-runtimes)ページから、kubeadmベースのセットアップでは`cgroupfs`ドライバーではなく、`systemd`ドライバーが推奨されると説明されています。
+
+このページでは、デフォルトで`systemd`ドライバーを使用して多くの異なるコンテナランタイムをセットアップする方法についての詳細も説明されています。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md:21]:
```suggestion
このページでは、デフォルトの`systemd`ドライバーを使用して多くの異なるコンテナランタイムをセットアップする方法についての詳細も説明されています。
```

File: content/ja/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+## {{% heading "prerequisites" %}}
+
+Kubernetesの[コンテナランタイムの要件](/docs/setup/production-environment/container-runtimes)を熟知している必要があります。
+
+<!-- steps -->
+
+## コンテナランタイムのcgroupドライバーの設定
+
+[Container runtimes](/docs/setup/production-environment/container-runtimes)ページから、kubeadmベースのセットアップでは`cgroupfs`ドライバーではなく、`systemd`ドライバーが推奨されると説明されています。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md:19]:
```suggestion
[Container runtimes](/docs/setup/production-environment/container-runtimes)ページでは、kubeadmベースのセットアップでは`cgroupfs`ドライバーではなく、`systemd`ドライバーが推奨されると説明されています。
```

**Review by nasa9084 — COMMENTED** [review]
/cc @riita10069

**Comment by ptux** [comment]
@inductor
@makocchi-git
@riita10069

Need lgtm to merge.
Could you take a look at the change?

**Review by riita10069 — APPROVED** [review]

**Pull request merged**